packages/api/src/controllers/user.ts

```diff
@@ -326,23 +355,55 @@ app.post("/", validatePost("user"), async (req, res) => {
 
 app.patch(
   "/:id/suspended",
+  validatePost("suspend-user-payload"),
   authMiddleware({ anyAdmin: true }),
   async (req, res) => {
+    const { suspended, emailTemplate } = req.body as SuspendUserPayload;
```

Comment:
Just a question, why is the emailTemplate sent as a body parameter instead of being defined somewhere in livepeer-com? I think the only one is `copyright` anyway, right?

Reply:
Yeah but maybe we don't want to send the email when we suspend, so the template is optional. And later/soon we might have other reasons to suspend the user so I thought it made sense to be more than just an opt-in boolean here.